src/pages/Touringcar/components/LocationsList/index.tsx

```diff
@@ -37,14 +48,21 @@ function LocationsList({ locationItems }: { locationItems: TouringcarStop[] }) {
       </TableHeaderStrong>
 
       <TableBody>
-        {sortedLocations.map(({ properties: item }, index) => {
-          if (!item) return <Fragment key={index}></Fragment>
+        {sortedLocations.map((feature, index) => {
+          const { properties } = feature
+
+          if (!feature || !properties) return <Fragment key={index}></Fragment>
 
           return (
-            <TableRow key={item.omschrijving}>
-              <TableCell>{item.omschrijving}</TableCell>
-              <TableCell width="25%">{item.plaatsen}</TableCell>
-            </TableRow>
+            <TableRowClickable
+              key={properties.omschrijving}
```

Comment:
Kun je voor deze key ook niet beter de index gebruiken? Waarom gebruik je hier een 'omschrijving', die mogelijk niet uniek is?

Reply:
Het zou het beste zijn om de `id` te gebruiken, maar mogelijk bestaat die niet (weet ik niet meer uit m'n hoofd). Ik ga nog even kijken naar wat de beste prop is.

Indices gebruiken als keys is een anti-pattern: https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318

Reply:
Aangepast